packages/browser/__tests__/login/oidc/redirectHandler/AuthCodeRedirectHandler.spec.ts

```diff
@@ -281,37 +310,56 @@ describe("AuthCodeRedirectHandler", () => {
     // We use ts-ignore comments here only to access mock call arguments
     /* eslint-disable @typescript-eslint/ban-ts-comment */
     it("returns an authenticated bearer fetch by default", async () => {
+      /* tslint:disable-next-line:no-any */
+      (global as any).localStorage = new LocalStorageMock();
```

Comment:
Does it work if this is replaced by `window.localStorage = ...` ?

Reply:
Yeah, that's better (although still need `(window as any)` to avoid TS error with overwriting a readonly property!) - 39ae8d7